src/crwimage_int.hpp

```diff
@@ -20,7 +20,7 @@ namespace Exiv2::Internal {
 class CiffHeader;
 class CiffComponent;
 struct CrwMapping;
-using CrwSubDir = struct {
+struct CrwSubDir {
   uint16_t dir;
```

Comment:
Unbelievable. This section actually fixes compilation on 32-bit ARM on Debian 11 with GCC 9 and 10. I have no idea why. The error is

```
../src/crwimage_int.hpp:239:16: warning: ‘virtual void Exiv2::Internal::CiffComponent::doRemove(Exiv2::Internal::CrwDirs&, uint16_t)’ used but never defined
  239 |   virtual void doRemove(CrwDirs& crwDirs, uint16_t crwTagId);
      |                ^~~~~~~~
../src/crwimage_int.hpp:237:26: warning: ‘virtual Exiv2::Internal::CiffComponent* Exiv2::Internal::CiffComponent::doAdd(Exiv2::Internal::CrwDirs&, uint16_t)’ used but never defined
```

That is just weird